react-hot-button-component/justin-app/src/HotButton.js

```diff
@@ -0,0 +1,27 @@
+import { useState } from "react";
+import './HotButton.css';
+
+export default function HotButton() {
+  const [clicked, setClicked] = useState(0)
+
+  function handleClick() {
+    setClicked(clicked + 1);
+  }
+
+  switch (true) {
```

Comment:
This works, but it duplicates the HTML repeatedly. With React components, it's usually best to have the HTML appear just once in a single `return`.

Note that you repeat the same code over and over again with the only difference being the className. See if you can figure out how to compute the className once and avoid all the duplicated code.